Libraries/Text/TextStylePropTypes.js

```diff
@@ -41,11 +41,14 @@ var TextStylePropTypes = Object.assign(Object.create(ViewStylePropTypes), {
     ['auto' /*default*/, 'left', 'right', 'center', 'justify']
   ),
   /**
-   * @platform ios
+   * Specifies the style of text decoration line; an underline, a line-through
+   * (often referred to as a strike-through), or both.  Can be specified as either a
+   * space-separated string of underline and line-through, or an array of the same.
    */
-  textDecorationLine: ReactPropTypes.oneOf(
-    ['none' /*default*/, 'underline', 'line-through', 'underline line-through']
-  ),
+  textDecorationLine: ReactPropTypes.oneOfType([
+    ReactPropTypes.oneOf(['none' /*default*/, 'underline', 'line-through', 'underline line-through']),
+    ReactPropTypes.arrayOf(ReactPropTypes.oneOf(['none', 'underline', 'line-through']))
```

Comment:
Why do you need the `arrayOf` here? Looks like the line above already has all the combinations?